main.js

```diff
@@ -0,0 +1,111 @@
+const express = require('express')
```

Comment:
curious why you chose to use js instead of ts here

Reply:
The new commit takes care of this. JS was used to quickly get the service up and running keeping time constraint in mind.